deps/chakrashim/src/v8functiontemplate.cc

```diff
@@ -237,10 +237,11 @@ class FunctionTemplateData : public TemplateData {
         return nullptr;
       }
 
+      Local<String> className = !instanceTemplate.IsEmpty() ?
+        instanceTemplate->GetClassName() : Local<String>();
```

Comment:
I need to check for `className` to add it to the `prototype` object, so instead of fetching it twice I just moved it out of the more restrictive scope.